cerberus/cerberus.py

```diff
@@ -454,6 +454,7 @@ def __normalize_mapping(self, mapping, schema):
         if self.purge_unknown:
             self._normalize_purge_unknown(mapping, schema)
         self._normalize_coerce(mapping, schema)
+        self._normalize_default(mapping, schema)
```

Comment:
> but maybe it's be useful to coerce a value to None in order to replace it with a default.

But as told by @nicolaiarocci and I also think the same, using default to replace `None` seems odd, specially if `None` is a valid value (`nullable` fields for example). But I know that maybe becomes handy, that's why I proposed to only override `None`s with default only for non `nullable` fields, but the @nicolaiarocci argument in the comment below seems valid too.

> coercing defaults on the other hand shouldn't make sense.

It shouldn't but I don't know if it hurts, maybe it's handy and safer.